fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/FedoraToOCFLObjectIndexImpl.java

```diff
@@ -35,10 +42,30 @@
  */
 @Component
 public class FedoraToOCFLObjectIndexImpl implements FedoraToOCFLObjectIndex {
+
     private static Logger LOGGER = LoggerFactory.getLogger(FedoraToOCFLObjectIndexImpl.class);
 
     private Map<String, FedoraOCFLMapping> fedoraOCFLMappingMap = Collections.synchronizedMap(new HashMap<>());
 
+    /**
+     * Constructor
```

Comment:
Please add a documentation note describing the expected layout/fields in the .tsv file.
The use of `map.length == 3` and indexing into the `map` array seem a bit opaque.